core/XH.ts

```diff
@@ -336,6 +337,27 @@ export class XHApi {
         this.acm.renderApp(appSpec);
     }
 
+    /**
+     * Entry-point to start the Hoist Admin console app, with common properties defaulted.
+     * Call this from within your project's `/client-app/src/apps/admin.ts` file.
+     *
+     * NOTE you must still import and pass in the `componentClass`, `containerClass`, and
+     * `modelClass` options. We don't default those here, as we do not want to import admin-only
+     * code in XH, where it would be added to the bundles for all client apps.
+     */
+    renderAdminApp<T extends HoistAppModel>(
+        appSpec: SetOptional<AppSpec<T>, 'isMobileApp' | 'checkAccess'>
+    ) {
+        this.acm.renderApp({
+            clientAppCode: 'admin',
+            clientAppName: `${this.appName} Admin`,
+            isMobileApp: false,
+            webSocketsEnabled: true,
```

Comment:
Realizing this might not be appropriate to default due to need to opt-in on server - can check that tomorrow